src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,31 +9,76 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+ }
   componentWillMount() {
     let { setCurrentPursuance, match, currentPursuanceId } = this.props;
     currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
     setCurrentPursuance(currentPursuanceId);
+    ReactModal.setAppElement('body');
+  }
+  componentDidMount(){
+   this.setState({modalIsOpen : true});
```

Comment:
@greg-han Perhaps this should be `this.openmodal()`?

Reply:
I figured direct state access is better than doing a function call because it's one less reference you are using for that specific process. I thought it was also "reactful" since react prefers states?
However, I totally trust your judgement. Can you please explain why a function call is better in this case? Is it for consistency across the program and good practice to use the same "thing" or function in this case for every action?